crt_portal/cts_forms/tests.py

```diff
@@ -55,17 +53,24 @@ def test_Contact_valid(self):
         })
         self.assertTrue(form.is_valid())
 
+    def test_Class_valid(self):
```

Comment:
Should we add a test for the invalid path as well, to make sure the form doesn't mark all protected class data as valid?

Reply:
The invalid test case is below: 
https://github.com/usdoj-crt/crt-portal/pull/91/files#diff-18509124e855af21de52f53726ce2005R66

Reply:
Ah, I see. I think I was confused by the difference between the `Valid_Form_Tests` and `Validation_Form_Tests` classes. Thanks!